src/python/pants/backend/python/target_types.py

```diff
@@ -240,11 +240,13 @@ class PexPlatformsField(StringSequenceField):
     alias = "platforms"
     help = (
         "The platforms the built PEX should be compatible with.\n\nThis defaults to the current "
-        "platform, but can be overridden to different platforms. You can give a list of multiple "
-        "platforms to create a multiplatform PEX.\n\nTo use wheels for specific "
-        "interpreter/platform tags, you can append them to the platform with hyphens like: "
-        'PLATFORM-IMPL-PYVER-ABI (e.g. "linux_x86_64-cp-27-cp27mu", '
-        '"macosx_10.12_x86_64-cp-36-cp36m"):\n\n  - PLATFORM: the host platform, e.g. '
+        "platform, but can be overridden to different platforms. There must be built wheels "
+        "available for all of the foreign platforms, rather than sdists.\n\n"
+        "You can give a list of multiple platforms to create a multiplatform PEX, "
+        "meaning that the PEX will be executable in all of the supported environments.\n\n"
+        "Platforms should be in the format defined by PEP 425, i.e. PLATFORM-IMPL-PYVER-ABI "
```

Comment:
The format is decidedly not PEP-425 compatible: https://www.python.org/dev/peps/pep-0425/#use

Reply:
Fix here: #12320